### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/server-platforms/laravel/01-login.md [hunk 1]
```diff
@@ -12,9 +12,10 @@ description: This tutorial will show you how to use the Auth0 PHP Laravel SDK to
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'laravel-auth0-sample',
+  pkgRepo: 'auth0-laravel-php-web-app',
+  githubUrl:'https://github.com/auth0-samples/auth0-laravel-php-web-app/tree/master/00-Starter-Seed';
```
Comment (mpaktiti) [inline articles/server-platforms/laravel/01-login.md]: @mehreencs87 The `;` at the end of the line throws a compilation error, please replace with `,`.
Also, you are missing the `pkgOrg` variable.

**Comment by mehreencs87** [comment]
@mpaktiti check the latest commit. Sorry for the inconvenience.

**Comment by mpaktiti** [comment]
Fixed by https://github.com/auth0/docs/pull/2078

**Pull request closed without merging**